interfaces/HO-Portal/src/app/user/user.page.html

```diff
@@ -133,19 +186,34 @@ <h1 class="text-uppercase">
             </div>
           </form>
         </div>
-        <div
-          *ngIf="passwordChanged"
-          class="ion-text-center"
-        >
-          <ion-icon
-            name="checkmark-circle"
-            color="success"
-            size="large"
-            aria-hidden="true"
-          ></ion-icon>
-          <br />
-          <strong>{{ 'page.user.change-password.success'|translate }}</strong>
-        </div>
+        <ng-container *ngIf="passwordChanged">
```

Comment:
Referring to discussion in the chat: was this design already discussed/aligned with Tijs or Esra?
- How it looks exactly
- That you see a different screen for a few seconds and then automatically go back (why not an alert/popup/etc?)